info-compiler/src/main/java/com/google/sps/infocompiler/InfoCompiler.java

```diff
@@ -160,6 +160,31 @@ public String handleResponse(final HttpResponse response) throws IOException {
     return json;
   }
 
+  /** 
+   * For testing purposes.
+   */
+  public JsonObject queryCivicInformation(String queryUrl, CloseableHttpClient httpclient)
```

Comment:
You shouldn't be writing a new method that is a duplicate of the real one just for testing purposes. Instead of testing private methods, you should be testing the end-to-end behavior that involves calling the private method itself.

Reply:
Got it, thanks!

Reply:
I changed this testing method and then re-wrote the bigger method queryCivicInformation().